docs/fhir/fhir_import_config.rst

```diff
@@ -128,26 +128,40 @@ Configuring related resources
 If a FHIR Import resource type has "Import related only" checked, we
 need to configure how the resource type is related.
 
-Navigate to FHIR > JSON Path to resource types, and click "Add JSON Path
-to resource type".
+For example, Patient resources may have associated ServiceRequests, which you
+may want to import. To achieve this, you'd set up a FHIRImportResourceType for
+ServiceRequests, with "import related only" checked, so only those associated
+with patients in the import are fetched.
 
-A ServiceRequest.subject is a reference to the Patient it is referring.
+Navigate to FHIR > "Resource type relationships", and click "Add
+resource type relationship".
 
-Set "Resource type" to "ServiceRequest".
+Relationships are configured on the child resource type, with a
+reference to their parent, just like a typical foreign key relationship.
+Options available in this config are:
 
-Set "JSONPath" to "$.subject.reference".
+* Resource type - the type of the child or descendant resource.
+* Jsonpath - the path used to identify the parent or ancestor resource.
+* Related resource type - the type of the parent or ancestor resource.
+* Related resource is parent - checkbox indicating whether CommCare
+  should set up a parent/child index between the two cases.
 
-Set "Related resource type" to "Patient".
+.. admonition:: Example: Adding a patient service request
```

Comment:
This is how you do an inset block with a custom title - no idea why that's called an "admonition", nor why "example" isn't one of the default block types.